app/models/sent_scheduled_message.rb

```diff
@@ -0,0 +1,11 @@
+class SentScheduledMessage < ActiveRecord::Base
+  belongs_to :employee
+  belongs_to :scheduled_message
+
+  validates :employee, presence: true, uniqueness: { scope: :scheduled_message }
+  validates :message_body, presence: true
```

Comment:
This isn't truly necessary because the entire message reference is stored here, but consider adding `:message_title`